README.md

```diff
@@ -172,6 +172,67 @@ If you use a value other than `$(PRODUCT_BUNDLE_IDENTIFIER)` in the `CFBundleURL
 
 > For more info please read the [React Native docs](https://facebook.github.io/react-native/docs/linking.html).
 
+#### Expo (EAS / SDK 41+)
+
+Open your app's `app.json` and add the following (note the second parameter with the empty object ({}) in the array is the plugin config):
+
+```
+  "expo": {
+    plugins: [['react-native-auth0', {}]],
+    ...
+  }
+```
+
+In order for android builds to work, you will need to specify an auth0 domain.
+There are two ways to do this with the config plugin:
+
+**With environment variable**:
+
+Set `EXPO_AUTH0_DOMAIN` or `EXPO_AUTH0_DOMAIN_ANDROID` in your eas.json (see: [Expo EAS Docs](https://docs.expo.dev/build-reference/variables/)) to set the values required in the build.gradle.
+
+This can be useful when you have multiple tenants that correspond with each EAS build target.
+
+**With app.json**
+
+Open your app's `app.json` and add the following under the "expo" key:
```

Comment:
Also possibly to dynamically change the domain via `app.config.js` btw, I know that sometimes environment variables in React Native can be hard to work with.